### ISSUE TITLE
`no-duplicates` false positive

### ISSUE THREAD

**Issue by NickHeiner** [benmosher/eslint-plugin-import#2115]
This is flagged as an error:

```ts
import type Registration from './Registration';
import type { RouteResolver } from './Registration';
```

However, if we combine it, we get a TS error:
```ts
import type Registration, { RouteResolver } from './Registration';
```

```
A type-only import can specify a default import or named bindings, but not both.ts(1363)
```

This combined form with both what the autofixer produces, and I what I would manually think should fix this.

On version `2.23.4`

**Comment by NickHeiner** [comment]
Actually, I think this is probably a duplicate of #2114.
